content/en/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Migrating telemetry and security agents from dockershim
+content_type: task 
+reviewers:
+- SergeyKanzhelev
+- jrcamp # for SignalFx (Splunk)
+weight: 70
+---
+
+<!-- overview -->
+
+With kubernetes 1.20 dockershim was deprecated. From the
+[Dockershim Deprecation FAQ](https://kubernetes.io/blog/2020/12/02/dockershim-faq/)
+you might already know that most apps do not have a direct dependency on runtime hosting
+containers. However, there are still a lot of telemetry and security agents
+that has a dependency on docker to collect containers metadata, logs and
+metrics. This document aggregates information on how to detect these
+dependencies and links on how to migrate these agents to use generic tools or
```

Comment:
nit: Could remove: 
"However, there are still a lot of telemetry and security agents ..."

Is this the focus of the page?
This document describes:
-  How to detect the dependency of `dockershim` in telemetry, logging, and security agents.
-  How to update telemetry, logging, security agents in your cluster.